expression/builtin_compare_vec.go

```diff
@@ -590,11 +590,79 @@ func (b *builtinNullEQStringSig) vecEvalInt(input *chunk.Chunk, result *chunk.Co
 }
 
 func (b *builtinLTIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinLTIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	var err error
+	var buf0, buf1 *chunk.Column
+	buf0, err = b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf0)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf0); err != nil {
+		return err
+	}
+	buf1, err = b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf1)
+	if err := b.args[1].VecEvalString(b.ctx, input, buf1); err != nil {
+		return err
+	}
+
+	result.ResizeInt64(n, false)
+	VecCompareInt(mysql.HasUnsignedFlag(b.args[0].GetType().Flag), mysql.HasUnsignedFlag(b.args[1].GetType().Flag), buf0, buf1, result)
+	result.MergeNulls(buf0, buf1)
+	vecCompareNull(buf0, buf1, result.Int64s())
+	vecResOfLT(result)
+	return nil
+}
+
+func vecResOfLT(result *chunk.Column) {
+	res := result.Int64s()
+	n := len(res)
+	for i := 0; i < n; i++ {
+		if result.IsNull(i) {
```

Comment:
This is not needed for LT?

Reply:
I think is need for LT. If either operand is null, the result is 0.

![image](https://user-images.githubusercontent.com/13314548/66905461-7a2b3e80-f038-11e9-85a6-5903482c8413.png)

Reply:
isNull is checked for LT. But not the `0`

Reply:
You mean do not need to check null? Like that:
![image](https://user-images.githubusercontent.com/13314548/66907407-de9bcd00-f03b-11e9-8a70-f39a6f0ef402.png)

Reply:
When i do not check null, like above, then unit test is ok, and the performance is improved. That means do not check null is OK ? 

```
BenchmarkVectorizedBuiltinCompareFunc/builtinLTIntSig-VecBuiltinFunc-8         	  300000	      3881 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinCompareFunc/builtinLTIntSig-NonVecBuiltinFunc-8      	   30000	     45429 ns/op	       0 B/op	       0 allocs/op
```

Reply:
I had remove the check null logic. And change 'vecResOfLT(result *chunk.Column)'  to 'func vecResOfLT(res []int64)'.  @XuHuaiyu  PTAL